hades2/index.md

```diff
@@ -73,6 +73,19 @@ Read how to create a powsybl bundle based on [itools](http://powsybl.github.io/d
 or on the [Grid Study Environment](http://powsybl.github.io/docs/installation/javafx-packager.html), a JavaFX desktop
 application.
 
+The itools-packager plugin of powsybl-core will copy all the maven dependencies to the share/java folder of the distribution. To enable a feature, you have to add a runtime dependency to the pom.xml file. If you do not want to modify the pom.xml file, you have to download the dependencies for Hades2 integration in the share/java folder of the itools distribution. The dependencies are available on Maven Central following that [link](https://mvnrepository.com/artifact/com.rte-france.powsybl). The following jar files have to be downloaded:
```

Comment:
For homogeneity, plugin names, file names and paths must be into quote as `this`: (here we have `itools-packager`, `pom.xml` and `share/java`).

Also maybe insert a line break between
> To enable a feature, you have to add a runtime dependency to the `pom.xml` file.

and

> If you do not want to modify the `pom.xml` file, [...]

to increase readability